examples/platformvm/getCurrentSupply.ts

```diff
@@ -20,7 +23,8 @@ const InitAvalanche = async () => {
 const main = async (): Promise<any> => {
   await InitAvalanche()
 
-  const currentSupply: BN = await pchain.getCurrentSupply()
+  const currentSupply: GetCurrentSupplyResponse =
+    await pchain.getCurrentSupply()
   console.log(currentSupply.toString())
```

Comment:
```suggestion
  console.log(currentSupply)
```
makes sense to print the object now that it's an interface